config/config.go

```diff
@@ -361,6 +361,7 @@ type OpenIdConfig struct {
 // DeploymentConfig provides details on how Kiali was deployed.
 type DeploymentConfig struct {
 	AccessibleNamespaces []string `yaml:"accessible_namespaces"`
+	ClusterWideAccess    bool     `yaml:"cluster_wide_access,omitempty"`
```

Comment:
I believe with `omitempty` a bool set to `false` will be omitted.   As long as unset would always reflect `false` I guess this is OK, but given the semantics of CWA I'm wondering if it's best to not omit.

Reply:
`omitempty` is meaningless in our context and not used. In fact, we could remove every mention of `omitempty` in this file and it would do nothing to us because it is only used for marshaling into YAML and AFAIK we do not marshal into yaml (we only unmarshal from the ConfigMap to the internal Go structs - we don't go the other way around and go from the structs to a YAML string - we certainly don't write back to the ConfigMap nor write to a yaml file anywhere).

See: https://pkg.go.dev/gopkg.in/yaml.v2#Marshal

```
omitempty    Only include the field if it's not set to the zero
             value for the type or to empty slices or maps.
             Zero valued structs will be omitted if all their public
             fields are zero, unless they implement an IsZero
             method (see the IsZeroer interface type), in which
             case the field will be excluded if IsZero returns true.
```

Reply:
It's fine if we're never going to marshal to YAML.  But in general, for booleans, I've been burned when using omitempty, so it was just a mention.